acceptance/tests/lookup/configuration_interpolation.rb

```diff
@@ -46,7 +48,7 @@ class some_mod {
       SITE
 
       on(master, "chmod -R 775 #{fq_tmp_environmentpath}")
-      on(master, "chmod -R 775 #{confdir}")
+      on(master, "chmod -R 775 #{master_confdir}")
```

Comment:
I know this test was written this way, but I don't like that it makes changes to the real `confdir`. Usually we create a temp confdir and point the master at that.